src/System.CommandLine.ApiCompatibility.Tests/ApiCompatibilityApprovalTests.System_CommandLine_api_is_not_changed.approved.txt

```diff
@@ -85,11 +80,13 @@ System.CommandLine
     public System.Collections.Generic.IReadOnlyList<Directive> Directives { get; }
     public System.Boolean EnablePosixBundling { get; }
     public System.Boolean EnableTokenReplacement { get; }
+    public System.IO.TextWriter Error { get; set; }
+    public System.IO.TextWriter Out { get; set; }
```

Comment:
The names `Error` and `Out` might need a little more context unless we're also renaming `CommandLineConfiguration`, e.g. `ConsoleOut` and `ConsoleError`.